driving-game/main.js

```diff
@@ -35,10 +35,13 @@ class Car {
   }
   start() {
     const car = this
-    setInterval(function () {
+    window.interval = setInterval(function () {
```

Comment:
Instead of storing the `interval` on the `window`, you should store it on the car instance (`this`).